test/lib/redis-store-spec.js

```diff
@@ -463,3 +466,39 @@ describe('defaults', function () {
   });
 });
 
+describe('wrap function', function () {
+
+  // Simulate retrieving a user from a database
+  function getUser(id, cb) {
+    setTimeout(function () {
+      cb(null, {id: id, name: 'Bob'});
+    }, 100);
+  }
+
+  var userId = 123;
+  var key = 'user_' + userId;
+  var fromCache = true;
+
+  it('should work with wrap function', function (done) {
```

Comment:
Oh ok that makes sense. I was just tricked by scope reach of the functions and point of the test. We should test for some of our cases with redis(store) and not if wrap is working or not as it is not the function we provide. I'll remove the comment related to this test.